test/mock_windows.uts

```diff
@@ -79,6 +77,36 @@ ifIndex DestinationPrefix                          NextHop
 
 test_read_routes6_windows()
 
+= Restore normal routes & Ifaces
+
+IFACES.reload()
+conf.route.resync()
+conf.route6.resync()
+
+# Loopback Adapter is not installed on Appveyor for an unknown reason...
```

Comment:
Right.
I'll leave the Net6 tests for when travis and appveyor support ipv6 (should be quite soon, at least for travis)